### DISCUSSION TITLE
changing login screen using plymouth.

### DISCUSSION THREAD

**Discussion by ronnie-1205** [JaKooLit/Hyprland-Dots#487] [Q&A]
i want to change the custom lockscreen that the install script installs and in place of that use a plymouth theme. right now, i have gotten the animation to work but the default login screen still appears.

**Comment by JaKooLit** [comment]
login screen is not the same as lockscreen

login screen, that would be the sddm theme

**Comment by ronnie-1205** [comment]
no ik that its just that the plymouth theme im using has a password prompt that is supposed to show but instead it always just shows the splash screen for 2 sec and then the garuda linux login screen comes... after i login, anytime i lock my laptop using ctrl+alt+l, it shows me the custom lock screen installed by the install script....

**Comment by JaKooLit** [comment]
ok so the question is...

you want to costumize lock screen am I correct?


edit ~/.config/hypr/hyprlock.conf
